<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>751</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General_2">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="label_case">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Case Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>671</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_description">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_des">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>671</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Market_2">
    <attribute name="title">
     <string>Market</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_daem">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Day ahead energy market</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_geography">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>PJM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Germany</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Australia-NSW</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_georaphy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Geography&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_horizontal_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>721</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_arbitrage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Arbitrage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_arbitrage_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Balancing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Technology</string>
    </attribute>
    <widget class="Line" name="line_horizontal_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>721</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_georaphy_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Type of Technology&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tech">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Energy Storage System</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electric Vehicle to Grid</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_e2p">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_case_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Energy to Power Ratio (kWh/kW)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>97</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>97</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
